src/mainboard/raptor-cs/talos-2/Makefile.inc

```diff
@@ -3,3 +3,5 @@
 bootblock-y += bootblock.c
 romstage-y += cbmem.c
 romstage-y += romstage.c
+romstage-y += ../../../soc/ibm/power9/istep_8_9.c
+romstage-y += ../../../soc/ibm/power9/istep_8_10.c
```

Comment:
These shouldn't be here, they are built because of `select CPU_IBM_POWER9` in Kconfig

Reply:
I have removed it